.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: ruby
+rvm:
+  - 2.2
+sudo: false
+cache:
+  bundler: false
+  directories:
+    - disque
+    - gems
+before_install:
+  - test -d disque/.git || git clone -n https://github.com/antirez/disque.git
```

Comment:
The disque directory is cached, so this should be faster because it only fetches new revisions and does a hard checkout.